renovate.json5

```diff
@@ -1,22 +1,18 @@
 {
-  extends: ['config:base', ':disableDigestUpdates'],
-  // Bump version ranges instead of pinning since we use lock files
-  rangeStrategy: 'bump',
-  // Set a status check pending for 3 days from release timestamp to guard against unpublishing
-  stabilityDays: 3,
+  extends: ['github>valora-inc/renovate-config:default.json5', ':disableDigestUpdates'],
+
   // Limit number of concurrent renovate branches/PRs, to avoid spamming the repo
   prConcurrentLimit: 2,
 
-  // Do not require manual approval from within the Dependency Dashboard issue before opening a PR.
-  // An alternative would be to require manual approval again, or to use specific packageRules
-  dependencyDashboardApproval: false,
+  // timezone for automerge schedule
+  timezone: 'America/Los_Angeles',
+
+  // Only automerge when the team is mostly offline
+  automergeSchedule: [
```

Comment:
in valora-inc/renovate-config this is set to *during* the workday, since backend repos have CD set up and we want them deploying during the workday. The default setting is inconvenient for wallet PR's since we want to minimize the need for rebasing during the workday, and don't have to worry about automated releases taking place at night without supervision